data/styles/clear/include/Basemap.mapcss

```diff
@@ -459,10 +459,14 @@ area|z15-16[amenity=parking][parking=lane],
 area|z15-16[amenity=parking][parking=street_side],
 {fill-color: none;}
 
+/* Other */
+area|z12-[amenity=police]
```

Comment:
Let's make it z17- just like the icon.

Re the #2856 idea
1. we haven't decided on it for sure (need more experimentation)
2. when/if we decide it'd be better to migrate all areas at once